<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,6,1">
        <property name="spacing">
         <number>18</number>
        </property>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>文件路径:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E_ProjectPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="B_ChooseNewProject">
          <property name="text">
           <string>选择</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icon/select.png</normaloff>../icon/select.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,0,3">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="L_ProjectName">
          <property name="text">
           <string>Sheet Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E_SheetName">
          <property name="text">
           <string>Gewichtliste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>项目名称:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E_ProjectName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,3,0,3">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>本表配置:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E_Configuration"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>数据日期:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="E_Date"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="B_Load">
          <property name="text">
           <string>Load </string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icon/import.png</normaloff>../icon/import.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>40</number>
         </property>
         <item row="1" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>厚度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Thickness">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>特征零件号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_ShortPartName">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>B</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>PR-Nr.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_PRNr">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>J</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>德文名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_DeutschName">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>下级零件数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_ChildrenAmount">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>I</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Pfeil</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Pfeil">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>H</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>零件号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_PartName">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="5,5">
           <item>
            <widget class="QLabel" name="Label_2">
             <property name="text">
              <string>分类</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Class">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>L</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>镀层</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Cladding">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>S</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>重量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Weight">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>N</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>中文名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_CNName">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>供应商</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Supplier">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>V</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="5,5">
           <item>
            <widget class="QLabel" name="Label">
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Set">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>K</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>平台-Kz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_PlatformKz">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>材料</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Material">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Q</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="5,5">
           <item>
            <widget class="QLabel" name="Label_3">
             <property name="text">
              <string>数量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Amount">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>M</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="5,5">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>强度等级</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="E_Strength">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>T</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
